packages/synapse-react-client/src/components/DialogBase.tsx

```diff
@@ -102,19 +102,22 @@ export const DialogBase = ({
       open={open}
       className={className}
       onClose={() => onCancel()}
-      sx={sx}
-      {...DialogProps}
-      PaperProps={{
-        sx: theme => ({
-          [theme.breakpoints.down('sm')]: {
-            margin: 0,
-            width: '100%',
-            height: '100%',
-            maxHeight: 'unset',
-            padding: '16px',
+      sx={theme => ({
+        ...sx,
+        [theme.breakpoints.down('sm')]: {
+          width: '100vw',
+          '.MuiDialog-container > .MuiPaper-root': {
+            [theme.breakpoints.down('sm')]: {
+              padding: '33px',
+              margin: 0,
+              width: '100%',
+              height: '100%',
+              maxHeight: 'unset',
+            },
```

Comment:
Is this separate wrapper necessary? You're already inside a `theme.breakpoints.down`
```suggestion
            padding: '33px',
            margin: 0,
            width: '100%',
            height: '100%',
            maxHeight: 'unset',
```

Reply:
Whoops, no, that is a mistake!